### PR TITLE
Hello, world!

### PR DESCRIPTION
**Pull request by bugron**
This PR is work in progress, so it is **not ready** to be merged yet. Please, review the translation and let me know if there is anything that should be changed.

- [x] Also translate tasks and solutions

### CODE REVIEW

File: 1-js/02-first-steps/01-hello-world/article.md [hunk 1]
```diff
@@ -1,87 +1,87 @@
...
+Ձեռնարկի այս մասը վերաբերում է բուն JavaScript լեզվին։
 
-But we need a working environment to run our scripts and, since this book is online, the browser is a good choice. We'll keep the amount of browser-specific commands (like `alert`) to a minimum so that you don't spend time on them if you plan to concentrate on another environment (like Node.js). We'll focus on JavaScript in the browser in the [next part](/ui) of the tutorial.
+Մեզ հարկավոր է աշխատանքային միջավայր, որտեղ գործարկելու ենք մեր սկրիպտերը։ Քանի որ այս ձեռնարկը առցանց է, ապա զննիչը լավ ընտրություն է։ Զննիչներին հատուկ հրամանները (օրինակ՝ `alert`) կօգտագործենք հնարավորինս քիչ, որպեսզի դրանց վրա շատ ժամանակ չծախսեք, եթե պլանավորում եք այլ միջավայրում աշխատել (օրինակ՝ Node.js)։ Մենք կկենտրոնանանք զննիչներում JavaScript–ի աշխատանքին ձեռնարկի [հաջորդ բաժնում](/ui)։
 
-So first, let's see how we attach a script to a webpage. For server-side environments (like Node.js), you can execute the script with a command like `"node my.js"`.
+Սկզվում տեսնենք, թե ինչպես էջին կցել որևէ սկրիպտ։ Սերվերային միջավայրերում (օրինակ՝ Node.js) սկրիպտը կարելի է գործարկել հետյալ կամ նմանատիպ հրամանով՝ `"node my.js"`։
```
Comment (ArthurHakobyan) [inline 1-js/02-first-steps/01-hello-world/article.md]: Սկզվում -> Սկզբում

**Comment by bugron** [comment]
@ArthurHakobyan this PR is ready for a full review.

**Review by ArthurHakobyan — APPROVED** [review]

**Review by ArthurHakobyan — APPROVED** [review]

**Comment by bugron** [comment]
Ready to be merged

**Pull request merged**